src/style.css

```diff
@@ -1 +1,11 @@
-/* styles go here */
+.next {
+  margin-top: 10px;
+}
+
+.frame {
+  margin-bottom: 20px;
+}
+
+.first {
+  margin-top: 0;
```

Comment:
it does nothing, so you can remove this class from CSS and HTML

or even more (optionally):
instead of wrapping all labels (the only `div` wrapper you need is for the radio inputs) into `div` you may add a class for `label` and set the margin directly for the label
P.S. set `display: block` for label to make it full-width

Also, it's good practice to set _only_ margin-top or _only_ margin-bottom